components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/service/CertificateValidationManagementService.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.x509Certificate.validation.service;
+
+import org.wso2.carbon.identity.x509Certificate.validation.exception.X509ConfigurationException;
+import org.wso2.carbon.identity.x509Certificate.validation.model.CACertificate;
+import org.wso2.carbon.identity.x509Certificate.validation.model.CACertificateInfo;
+import org.wso2.carbon.identity.x509Certificate.validation.model.Validator;
+
+import java.security.cert.X509Certificate;
+import java.util.List;
+
+/**
+ * This interface supports the x509 authenticator validator manager.
+ */
+public interface CertificateValidationManagementService {
+
+    /**
+     * Get validators.
+     *
+     * @param tenantId Tenant Id.
+     * @return List of validators.
+     * @throws X509ConfigurationException If an error occurs while getting the validators.
+     */
+    List<Validator> getValidators(int tenantId) throws X509ConfigurationException;
```

Comment:
Usually in service level are dealt with the `tenantDomain`, not with the `tenantId`. Do we have a blocker to use `tenantDomain` in the service level?

Reply:
no will use it

Reply:
fixed with https://github.com/wso2-extensions/identity-x509-commons/pull/57/commits/52f92488d265780c25c0d23c5ec638ca00bd3f59